test/forms/BusinessNameFormSpec.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms
+
+import forms.BusinessNameForm._
+import play.api.data.FormError
+import utils.TestUtil
+
+class BusinessNameFormSpec extends TestUtil {
+
+  "The business name form" should {
+
+    val maxLengthErrorMessage: String = "captureBusinessName.error.exceedsMaxLength"
+    val noEntryErrorMessage: String = "captureBusinessName.error.empty"
+    val unchangedErrorMessage: String = "captureBusinessName.error.notChanged"
+
+    val testBusinessName = "Valid Business Name"
+
+    "validate that testBusinessName is valid" in {
+      val actual = businessNameForm("").bind(Map("business-name" -> testBusinessName))
+      actual.value shouldBe Some(testBusinessName)
+    }
+
+    "validate that testBusinessName capitalised is valid" in {
+      val actual = businessNameForm("").bind(Map("business-name" -> testBusinessName.toUpperCase()))
+      actual.value shouldBe Some(testBusinessName.toUpperCase())
+    }
```

Comment:
What is this supposed to be testing? It looks like it's just testing the toUpperCase method which isn't our code